### PR TITLE
added counter data UI

### LINKED ISSUE #254: Feature Request: Add live counter
### Is your feature request related to a problem? Please describe.

I wish to add an animated counter below the contributors in the contributor section which shows the no. of contributors and no. of pull requests etc. to add more liveliness to the site

### Describe the solution you'd like.

something similar to this.

image

### Describe alternatives you've considered

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by wasp0094**
Signed-off-by: wasp0094 <90551491+wasp0094@users.noreply.github.com>

# Pull Request Template

## Description

Added static data for the animated counter and animated hover effect in the contributors component. It is responsive on small screens and supports dark mode.

Fixes #254 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality).

## Additional Context (Please include any Screenshots/gifs if relevant)

Screenshot (179)
light mode.

Screenshot (180)
dark mode.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Review by padmajabhol — CHANGES_REQUESTED** [review]
Can we improve the UI a bit? Not talking about the hover effect here. Maybe we can make it similar to how it is in the sponsor's section.

Also, keep some padding on the top.

**Comment by wasp0094** [comment]
> Can we improve the UI a bit? Not talking about the hover effect here. Maybe we can make it similar to how it is in the sponsor's section.
> 
> Also, keep some padding on the top.

sure, I'll see to it 👍

**Comment by wasp0094** [comment]
improved the UI as requested, it now looks like this:
Screenshot (181)
dark mode.

Screenshot (182)
light mode.

**Review by padmajabhol — CHANGES_REQUESTED** [review]
The data isn't accurate. Change 30+ contributors to 40+, forks to 80+ , 120+ PRs,  40+ stars.

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hey @wasp0094 , great work with the UI. I see that the stars, contributors, PRs etc. data are static content and aren't updated automatically.

Can you come up with dynamic updation of this data via API or some similar technique?

**Comment by padmajabhol** [comment]
You can use the Github API to make the calls, though I think this will require parsing. Considering you are a beginner, you can always ask @sohamsshah or me on how to work through this issue if you aren't sure or we can create another issue to make the data dynamic.  For reference : https://gist.github.com/jasonrudolph/6057563

File: website/src/css/custom.css [hunk 1]
```diff
@@ -231,3 +255,95 @@ html[data-theme='dark'] .footer__link-item:hover {
...
+  }
+
+  .footer__col:last-of-type{
+    width: 35%;
+  }
+
+  .footer__col{
+    padding: 3%;
+  }
+}
```
Comment (padmajabhol) [inline website/src/css/custom.css:349]: Can you resolve the conflicts on this particular file?

**Comment by sohamsshah** [comment]
Hey @wasp0094 , any updates on this PR? Please feel free to reach out if you are facing any friction in implementing this.

**Comment by wasp0094** [comment]
hi, yes i'm still working on it and discussed it with @padmajabhol

**Comment by sohamsshah** [comment]
Awesome @wasp0094 ! 🚀

**Pull request closed without merging**